lib/CalDAV/Plugin.php

```
@@ -264,6 +264,8 @@ public function report($reportName, $report, $path)
 
                 return false;
         }
+
+        return true;
```

what to do about these sort of things?
I guess that this code is never supposed to get past the switch cases. But what if it does? return true or false?

---

Maybe throw a exception?

---

I don't really know the calling path of all these plugins. If `$reportName` does not match any of the switch cases, then maybe the function is supposed to return "nothing" and not explode?

Is the existing omission of a return at the end of the function a bug or a feature?

---

I got rid of this and updated the PHPdoc to allow null.